File: app/src/main/java/lt/gecius/test_dagger2/di/intoset/IntoSetComponent.java
package lt.gecius.test_dagger2.di.intoset;

import dagger.Component;
import lt.gecius.test_dagger2.app.IntoSetActivity;

@Component (modules = IntoSetModule.class)
public interface IntoSetComponent {

    void injectInto(IntoSetActivity activity);
}

File: app/src/main/java/lt/gecius/test_dagger2/di/intoset/IntoSetComponent.kt
package lt.gecius.test_dagger2.di.intoset

import dagger.Component
import lt.gecius.test_dagger2.app.IntoSetActivity

@Component(modules = arrayOf(IntoSetModule::class))
interface IntoSetComponent {

    fun injectInto(activity: IntoSetActivity)
}
